app/controllers/courses_controller.rb

```diff
@@ -25,15 +25,22 @@ def index
   def edit; end
 
   def update
+    if current_role.student? || current_role.ta?
```

Comment:
@WeissSa You can revert this change as well (since you've now removed the change to the corresponding test)

Reply:
Oops. Thought I did but must have forgotten to add it.